axelrod/strategies/__init__.py

```diff
@@ -29,3 +30,44 @@
 cheating_strategies = [s for s in all_strategies if not obey_axelrod(s())]
 
 ordinary_strategies = strategies  # This is a legacy and will be removed
+
+
+def filtered_strategies(filterset, strategies=all_strategies):
+    """
+    Applies the filters defined in the given filterset dict and returns those
+    strategy classes which pass all of those filters from the given list of
+    strategies.
+
+    e.g.
+
+    For the filterset dict:
+        {
+            'stochastic': True,
+            'min_memory_depth': 2
```

Comment:
Can we add an example of makes use of to the docstring, since it's a bit different than the others?